src/test/bridges/angle/Angle.t.sol

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec.
+pragma solidity >=0.8.4;
+
+import {BridgeTestBase} from "./../../aztec/base/BridgeTestBase.sol";
+import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {ICERC20} from "../../../interfaces/compound/ICERC20.sol";
+import {AngleBridge} from "../../../bridges/angle/AngleBridge.sol";
+import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
+
+contract AngleTest is BridgeTestBase {
```

Comment:
Please consider making two separate test files for unit and E2E tests.

Also, tests are only done with Dai, but you have multiple collaterals for the bridge, test more than once, especially when you have a different number of decimals across the assets.